### ISSUE TITLE
403 for Vimeo videos

### ISSUE THREAD

**Issue by hugovk** [sendgrid/docs#4227]
## Expected content

Playable Vimeo videos.

## Actual content

Unplayable Vimeo videos.

![image](https://user-images.githubusercontent.com/1324225/47024949-5904e900-d16b-11e8-9c0e-bb43a90a5d53.png)

**Link to page**:

https://sendgrid.com/docs/ui/account-and-settings/two-factor-authentication/

=>

https://github.com/sendgrid/docs/blob/develop/content/docs/ui/account-and-settings/two-factor-authentication.md

These videos in total:

https://player.vimeo.com/video/120703745
https://player.vimeo.com/video/120709155
https://player.vimeo.com/video/120738522
https://player.vimeo.com/video/121404093
https://player.vimeo.com/video/130900679
https://player.vimeo.com/video/221494705
https://player.vimeo.com/video/233700782
https://player.vimeo.com/video/248169751
https://player.vimeo.com/video/263354373
https://player.vimeo.com/video/265066856
https://player.vimeo.com/video/265136044
https://player.vimeo.com/video/265831363
https://player.vimeo.com/video/287678072

**Comment by ksigler7** [comment]
@hugovk - when I view this page, I am seeing the video:
![image](https://user-images.githubusercontent.com/28788585/47049210-e6d7d880-d159-11e8-8ecc-b9f41c8260f2.png)

Can you give us any info about your browser?

**Comment by hugovk** [comment]
I'm using macOS High Sierra and Chrome 69.

And I found the problem, its cookies are being blocked by Privacy Badger:

![image](https://user-images.githubusercontent.com/1324225/47134768-945ffe80-d2b6-11e8-86ce-cd84343b08f9.png)

Switching the Vimeo setting lets it work:

![image](https://user-images.githubusercontent.com/1324225/47134791-ab9eec00-d2b6-11e8-8d7d-8470fceb0e47.png)

I guess this is kind of a false alarm, but I don't know why it needs to require cookies to play a video.

And I used a command-line tool to check links, which also returns a 403. Minimal example:

```sh
curl "https://player.vimeo.com/video/120703745" \
--silent \
--location \
--write-out "%{http_code}" \
--output /dev/null
```

Feel free to close this, but if there are any settings at Vimeo to make it playable without cookies, that would be great.
